extensions/blocks/eventbrite/modal-button-preview.js

```diff
@@ -0,0 +1,165 @@
+ * (Using Gutenberg code that shipped with WordPress 5.3)
+ *
+ * @see https://github.com/WordPress/gutenberg/blob/wp/5.3/packages/block-library/src/button/edit.js
+ *
+ * Removes url for button, since the button triggers a modal in this block
+ * rather than opening a link.
+ */
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, useCallback } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { PanelBody, RangeControl, withFallbackStyles } from '@wordpress/components';
+import {
+	RichText,
+	ContrastChecker,
+	InspectorControls,
+	withColors,
+	PanelColorSettings,
+} from '@wordpress/block-editor';
+
+const { getComputedStyle } = window;
+
+const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { textColor, backgroundColor } = ownProps;
+	const backgroundColorValue = backgroundColor && backgroundColor.color;
+	const textColorValue = textColor && textColor.color;
+	//avoid the use of querySelector if textColor color is known and verify if node is available.
+	const textNode =
+		! textColorValue && node ? node.querySelector( '[contenteditable="true"]' ) : null;
+	return {
+		fallbackBackgroundColor:
+			backgroundColorValue || ! node ? undefined : getComputedStyle( node ).backgroundColor,
+		fallbackTextColor:
+			textColorValue || ! textNode ? undefined : getComputedStyle( textNode ).color,
+	};
+} );
+
+const MIN_BORDER_RADIUS_VALUE = 0;
+const MAX_BORDER_RADIUS_VALUE = 50;
+const INITIAL_BORDER_RADIUS_POSITION = 5;
+
+function BorderPanel( { borderRadius = '', setAttributes } ) {
+	const setBorderRadius = useCallback(
+		newBorderRadius => {
+			setAttributes( { borderRadius: newBorderRadius } );
+		},
+		[ setAttributes ]
+	);
+	return (
+		<PanelBody title={ __( 'Border Settings' ) }>
+			<RangeControl
+				value={ borderRadius }
+				label={ __( 'Border Radius' ) }
+				min={ MIN_BORDER_RADIUS_VALUE }
+				max={ MAX_BORDER_RADIUS_VALUE }
+				initialPosition={ INITIAL_BORDER_RADIUS_POSITION }
+				allowReset
+				onChange={ setBorderRadius }
+			/>
+		</PanelBody>
+	);
+}
+
+class ButtonEdit extends Component {
+	constructor() {
+		super( ...arguments );
+		this.nodeRef = null;
+		this.bindRef = this.bindRef.bind( this );
+	}
+
+	bindRef( node ) {
+		if ( ! node ) {
+			return;
+		}
+		this.nodeRef = node;
+	}
+
+	render() {
+		const {
+			attributes,
+			backgroundColor,
+			textColor,
+			setBackgroundColor,
+			setTextColor,
+			fallbackBackgroundColor,
+			fallbackTextColor,
+			setAttributes,
+			className,
+		} = this.props;
+
+		const { borderRadius, placeholder, text, title } = attributes;
+
+		return (
+			<div
+				className={ classnames( className, 'is-modal-button' ) }
+				title={ title }
+				ref={ this.bindRef }
+			>
+				<RichText
+					placeholder={ placeholder || __( 'Add text…' ) }
```

Comment:
Should we add `jetpack` as domain for all the translated strings in this file?

Reply:
Good catch. Yes, I think so. But I'm unclear on this--these are all copied directly from Core.

Reply:
I can see the reason to leave them without domain, but what would happen if Core stops translating those strings?
Honest question, I have no idea! I can guess we'd be left with these strings untranslated, but I'm not sure if there's any mechanism in place for backward compatibility.

@Automattic/i18n maybe y'all know more about this?
For context: in this block we had to recreate some UI _almost_ identical to Core.
The copy is the same so far though, so we added the strings without text domain, in order to reuse the Core's translations.
My fear is that if Core drops those strings, we'd end up with untranslated text.
On the other hand, if we add the `jetpack` domain, we'd need to translate again a bunch of strings that are already translated in Core.
What would you suggest to do?

Reply:
Please add the text domain.

> I can see the reason to leave them without domain, but what would happen if Core stops translating those strings?

I still need to read up on how jetpack is handling language packs, from what I know so far and going by core i18n docs I suspect the translations may stop loading.

Reply:
@lsl Added already! 😄